src/LinkDotNet.Blog.Web/Controller/RssFeedController.cs

```diff
@@ -77,17 +88,21 @@ private static XmlWriterSettings CreateXmlWriterSettings()
     private static SyndicationItem CreateSyndicationItemFromBlogPost(string url, BlogPostRssInfo blogPost)
     {
         var blogPostUrl = url + $"/blogPost/{blogPost.Id}";
-        var shortDescription = MarkdownConverter.ToPlainString(blogPost.ShortDescription);
+        var content = MarkdownConverter.ToMarkupString(blogPost.ShortDescription ?? blogPost.Content ?? string.Empty);
```

Comment:
I wouldn't handle `string.Empty` here and basically check before and maybe even throw an exception. That is a case we don't want.
`string.Empty` would probably lead to "swallowing" the issue.

Reply:
True! I'll update that!